src/Estimators/include/BipedalLocomotion/FloatingBaseEstimators/FloatingBaseEstimator.h

```diff
@@ -83,6 +83,21 @@ class FloatingBaseEstimator : public BipedalLocomotion::System::Advanceable<Floa
                            iDynTree::Transform& IMU_H_l_foot,
                            iDynTree::Transform& IMU_H_r_foot);
 
+        /**
+        * Get relative pose between IMU and the feet
+        * @param[in] encoders joint positions through encoder measurements
+        * @param[out] IMU_H_l_foot pose of the left foot contact frame with respect to the IMU frame
+        * @param[out] IMU_H_r_foot pose of the right foot contact frame with respect to the IMU frame
+        * @param[out] J_IMULF explicit Jacobian of left foot frame with respect to IMU frame
```

Comment:
What do you mean by "explicit" jacobian?

Reply:
I suppose I copied the actual KinDynComputation's method name where we have to "explicitly" specify the frame indices.
Usually, it's obtained wrt the base link, instead here, we have here wrt the IMU.

Reply:
Should I remove that?

Reply:
As you prefer, personally I don't think that the explicit meaning is clear in this context, so to avoid confusion it is better to remove it, but it is not blocking the merge of the PR for me.

Reply:
You are right! I updated the documentation https://github.com/dic-iit/bipedal-locomotion-framework/pull/85/commits/c5b51a1695e10c955ea54019e2d09c3ddb552274.